tests/functional/backend/SettingsPluginTest.php

```diff
@@ -39,17 +63,17 @@ public static function setUpBeforeClass(): void
 
         $obj = new \stdClass();
         $obj->customProperty = 'abc';
-       
-        self::$settings = [
+
+        self::$encryptedSettings = [
```

Comment:
Why have you changed this?
This was working just fine. It was an old test for testing get/set for settings and different types.

Reply:
I can revert the name, this is not the issue.

Reply:
Well, if renaming it, why not "settingValuesToBeTested"

Reply:
settingsValue directly then ? I just see it test value too … not only encrypted …

You already [rename some functions](https://github.com/LimeSurvey/LimeSurvey/pull/2753/commits/076c8e97f07ef22b1f02da8feaf3bb263cdecdff)  fixing language.

PS : we can set the plugins->settings too, but i don't think we have way to save object. The default are `'string'`

Reply:
>  I just see it test value too … not only encrypted …

It tests values are read/write ok both plain and encrypted.

> You already [rename some functions](https://github.com/LimeSurvey/LimeSurvey/pull/2753/commits/076c8e97f07ef22b1f02da8feaf3bb263cdecdff) fixing language.

Yes, small typo.

> PS : we can set the plugins->settings too, but i don't think we have way to save object. The default are 'string'

Let's leave as it is. Agree. If needed, we can change later.